packages/runtime/src/builder/serialization/controls/style-v2.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleV2ControlDefinition } from '../../../controls'
+import { Serialize, Deserialize } from './types'
+
+export function serializeStyleV2Control(
+  control: StyleV2ControlDefinition,
+): [Serialize<StyleV2ControlDefinition>, Transferable[]] {
+  const { getStyle, ...config } = control.config
+
+  const transferables: Transferable[] = []
+  return [
+    {
+      ...control,
+      config,
+    },
+    transferables,
+  ] as [Serialize<StyleV2ControlDefinition>, Transferable[]]
+}
+
+export function deserializeStyleV2Control(
+  control: Serialize<StyleV2ControlDefinition>,
+): Deserialize<Serialize<StyleV2ControlDefinition>> {
+  // TODO: `as unknown` shouldn't be required here 
```

Comment:
What you'll need to do here is compose the deserialize function for other controls. Take a look at how other composition controls implement deserialization